packages/build/src/npm-packages/publish.ts

```diff
@@ -35,26 +46,36 @@ export function publishNpmPackages(
         'from-package',
         '--no-private',
         '--no-changelog',
-        '--no-push',
         '--exact',
-        '--no-git-tag-version',
+        // During mongosh releases we handle the tags manually
+        ...(!isAuxiliaryOnly ? ['--no-git-tag-version', '--no-push'] : []),
```

Comment:
Hm, this does mean that when doing a mongosh release we'd only have 1 version tag for the whole release (and thus miss the auxiliary package bump tags) but when doing just a auxiliary bump we'd have all the aux. packages i.e. `browser-repl@3.1.0`.  I don't know if tags matter for us (largely adapting it from `compass`) but if we want tags to always exist for the auxiliary packages I could handle the tagging for those manually as well.

Reply:
I honestly don't see any harm in `mongosh@3.1.0` and `3.1.0` existing at the same time, so I could just have this get published in all cases